src/bin/asr_train.py

```diff
@@ -41,14 +41,10 @@ def main():
     parser.add_argument('--verbose', '-V', default=0, type=int,
                         help='Verbose option')
     # task related
-    parser.add_argument('--train-feat', type=str, required=True,
```

Comment:
For now, it's better to add error messages (ask the user to redo the "stage 2: Dictionary and Json Data Preparation") instead of removing these four options. Later, we can remove these options.